packages/ag-charts-community/src/scene/extendedPath2D.ts

```diff
@@ -299,48 +300,72 @@ export class ExtendedPath2D {
     }
 
     // https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d
-    computeSVGDataPath(ox: number, oy: number): string {
+    computeSVGDataPath(matrix: Matrix): string {
```

Comment:
I do not know whether this is correct approach.

My understanding is that we want to avoid using transformation matrices unless it's absolutely necessary. The `ExtendedPath2D` object should in theory have the rotation and scale transformations pre-computed before invoking `computeSVGDataPath` and only require an `{ox, oy}` translation.

Could you please provide more information about the issue this PR is solving? (e.g. ticket number, steps-to-repro, screenshot, ...)

Reply:
Discussed on Slack:
It has to do with markers which are scaleable and rotatable: https://github.com/ag-grid/ag-charts/blob/b96abab48840caf08dec8eba981ab7ee24cb7bec/packages/ag-charts-community/src/chart/marker/marker.ts#L58